models/topic-model.js

```diff
@@ -29,4 +29,36 @@ const selectUsers = () => {
   });
 };
 
-module.exports = { selectTopics, selectUsers, selectArticles, selectArticleById };
+const setArticleById = (inc_votes, article_id) => {
+  if (!inc_votes || typeof parseInt(article_id) != "number") {
+    return Promise.reject({
```

Comment:
if the article id were not a number, would PSQL now catch this and throw it's own error that we could catch with some error handling middleware